src/core/bot/DraftBot.ts

```diff
@@ -72,6 +76,36 @@ export class DraftBot {
 		setTimeout(DraftBot.dailyTimeout, millisTill);
 	}
 
+	/**
+	 * Send a notification every minute for player who arrived in the last minute
+	 */
+	async reportNotifications(this: void): Promise<void> {
+		const playersToNotify = <{ discordUserId: string }[]>(await draftBotInstance.gameDatabase.sequelize.query(`
```

Comment:
Alors, non. Tu vas surcharger tout en faisant ça car tu vas retourner beaucoup trop de joueur (ça coute cher en cpu et ram), tu vas méga surcharger la base avec ton for car c'est couteux un Players.getOrRegister, sur la base de prod ça va prendre un temps de fou. Tu ne peux pas te permettre de get des dizaines de milliers de joueurs par minutes.
Une solution est de faire une seule requête SQL qui récupère directement tous les joueurs qui ont fini leur trajet (globalement c'est start date + effect time (si > à start date) > temps du trajet). Une requête SQL seule c'est rapide et renvoyer une dizaines d'ids aussi